docs/build/messages/read-receipt.mdx

```diff
@@ -91,17 +91,10 @@ Read receipts for React Native haven't been implemented yet.
 
 ## Send a read receipt
 
-- `messageId`: ID of the message that was read
-- `timestamp`: Timestamp the read receipt sent, in ISO 8601 format
-
 <Tabs groupId="sdk-langs">
 <TabItem value="js" label="JavaScript"  attributes={{className: "js_tab"}}>
 
-```tsx
-const readReceipt: ReadReceipt = {};
-```
-
-Once you've created a read receipt, you can send it:
+The content of a read receipt message must be an empty object.
```

Comment:
@rygine - this is a very helpful statement! Just  curious - do it apply only to the JavaScript tab? Or should we move it right above all of the tabs?

Reply:
ahh, good call out! that statement was only for JavaScript and by extension, the React SDK.